package.json

```diff
@@ -3,19 +3,24 @@
   "version": "0.0.1",
   "description": "placeholder-app-description",
   "scripts": {
-    "postinstall": "npm run build-app",
-    "build-app": "cd app && npm install && cd ..",
+    "postinstall": "yarn compile && yarn copy-artifacts && yarn build-app",
+    "copy-artifacts": "cp -R node_modules/@aragon/abis/os/artifacts/. artifacts/.",
```

Comment:
Interesting, why is this necessary? It looks like we're still importing artifacts from `@aragon/abis` later.